examples/cms-sanity/components/cover-image.js

```diff
@@ -1,14 +1,22 @@
 import cn from 'classnames'
 import Link from 'next/link'
+import { imageBuilder } from '../lib/api'
 
 export default function CoverImage({ title, url, slug }) {
   const image = (
     <img
-      src={url}
+      width={2000}
+      height={1000}
       alt={`Cover Image for ${title}`}
       className={cn('shadow-small', {
         'hover:shadow-medium transition-shadow duration-200': slug,
       })}
+      style={{ maxWidth: '100%', maxHeight: '100%' }}
```

Comment:
Was this necessary? I think tailwind adds `maxwidth` and `maxHeight` by default:

```css
img,
video {
  max-width: 100%;
  height: auto;
}

```

https://unpkg.com/tailwindcss@1.2.0/dist/base.css

Reply:
Oh neat, didn't realize it did that automatically 👍